setup.py

```diff
@@ -2,6 +2,7 @@
 from setuptools import setup, find_packages
 
 from pyDeltaRCM import shared_tools
+import pyDeltaRCM
```

Comment:
Is it possible to call the `command_line:run_model` more directly than total package import? I'm afraid this line would fail for any number of reasons totally unrelated to the command line runner and it could be annoying to debug.

Reply:
I changed this to a specific import of `from pyDeltaRCM import command_line`.